Mathlib/Order/Bounds/Extra.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2025 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import Mathlib.Order.Bounds.Basic
+import Mathlib.Data.Set.Prod
+
+/-!
+# Extra results
+
+Further results for `upperBounds` and `lowerBounds`
+
+-/
+
+open Set
+
+universe u v
+
+variable {α : Type u} {β : Type v}
+variable [Preorder α] [Preorder β] {f : α → β}
+
+lemma Monotone.dominated (Hf : Monotone f) {s₁ s₂ : Set α} (h : Dominated (· ≤ ·) s₁ s₂) :
+    Dominated (· ≤ ·) (f '' s₁) (f '' s₂) := fun a ha => by
+  obtain ⟨c, hcs, hcfa⟩ := ha
+  obtain ⟨d, hdd, hcd⟩ := h c hcs
+  simp only [mem_image, exists_exists_and_eq_and]
+  rw [← hcfa]
+  exact ⟨d, ⟨hdd, Hf hcd⟩⟩
+
+lemma Monotone.dominated_geq (Hf : Monotone f) {s₁ s₂ : Set α} (h : Dominated (· ≥ ·) s₁ s₂) :
+    Dominated (· ≥ ·) (f '' s₁) (f '' s₂) := fun a ha => by
+  obtain ⟨c, hcs, hcfa⟩ := ha
+  obtain ⟨d, hdd, hcd⟩ := h c hcs
+  simp only [mem_image, exists_exists_and_eq_and]
+  rw [← hcfa]
+  exact ⟨d, ⟨hdd, Hf hcd⟩⟩
```

Comment:
Could you also add the `Antitone` version of both these lemmas?

Reply:
Done.